### PR TITLE
Support of UIPageControl.

### PR DESCRIPTION
**Pull request by yfujiki**
Added styling for UIPageControl (`pageIndicatorTintColor` and `currentPageIndicatorTintColor`). 

It was mostly straightforward, just followed implementation of other controls. But L27-32 of `NUIPageControlRenderer.m` was required because none of the specified color does not take effect otherwise. It seems new color for each dot takes effect only after once being set as current.

### CODE REVIEW

**Comment by Stunner** [comment]
Tests appear to be failing on Xcode 6.x due to the following:
```
Fetching: activesupport-5.0.0.beta1.1.gem (100%)
ERROR:  Error installing cocoapods:
	activesupport requires Ruby version >= 2.2.2.
The command "gem install cocoapods" failed and exited with 1 during .
```
@timbodeit Do you have a preferred way of resolving this issue? I can take a shot at tackling this but I have got to admit I am a novice with this whole Travis CI (and CI for that matter).

**Comment by timbodeit** [comment]
@Stunner See #325
I'm not exactly sure, why rubygems chose an incompatible beta version of activesupport, but installing via bundler works fine.

File: NUI/NUIStyle.nss [hunk 1]
```diff
@@ -94,6 +94,10 @@ SegmentedControl {
 Switch {
     on-tint-color: @primaryBackgroundTintColor;
 }
+PageControl {
+    color:@secondaryFontColor;
```
Comment (timbodeit) [inline NUI/NUIStyle.nss]: Inconsistent whitespace here ...

File: README.md [hunk 2]
```diff
@@ -376,6 +376,13 @@ Below are all of the currently available style classes, their corresponding UI c
 * text-shadow-offset *(Offset)*
 * title-vertical-offset *(Number)*
 
+#### PageControl
+
+*UIPageControl*
+
+* color *(Color)*   `pageIndicatorTintColor`
```
Comment (timbodeit) [inline README.md]: ... and here.

**Comment by timbodeit** [comment]
Awesome! LGTM.
Thanks for splitting everything up into small, understandable commits :+1:

Unit test coverage for this would be cool, but in my opinion this is good to merge once #325 is.

**Pull request merged**